linker/shared/src/main/scala/org/scalajs/linker/frontend/BaseLinker.scala

```diff
@@ -33,19 +33,19 @@ import Analysis._
 /** Links the information from [[interface.IRFile IRFile]]s into
  *  [[standard.LinkedClass LinkedClass]]es. Does a dead code elimination pass.
  */
-final class BaseLinker(config: CommonPhaseConfig) {
+final class BaseLinker(config: CommonPhaseConfig, checkIR: Boolean) {
```

Comment:
It seems it wouldn't cost anything to add a deprecated constructor with the old signature here (defaulting to `checkIR = false`).